meta-mender-commercial/recipes-mender/mender-binary-delta/mender-binary-delta_1.4.1.bb

```diff
@@ -0,0 +1,19 @@
+require mender-binary-delta.inc
+
+# In dunfell the tarball is extracted out of bitbake, explicitly add the licenses directory.
+SRC_URI_append = " file://licenses"
+
+# DO NOT change the checksum here without make sure that ALL licenses (including
+# dependencies) are included in the LICENSE variable below.
+LICENSE = "Mender-Yocto-Layer-License.md & Apache-2.0"
+LICENSE_FLAGS = "commercial_mender-yocto-layer-license"
+LIC_FILES_CHKSUM = " \
+    file://${WORKDIR}/licenses/LICENSE.md;md5=66a40d48ea33620d1bb8d9a4204cde36 \
+    file://${WORKDIR}/licenses/xdelta/xdelta3/LICENSE;md5=cf96fa0d649f7c7b16616d95e7880a73 \
+    file://${WORKDIR}/licenses/xdelta/xdelta3/cpp-btree/COPYING;md5=3b83ef96387f14655fc854ddc3c6bd57 \
```

Comment:
@kacf This is my suggestion: to add `${WORKDIR}` here.

I've been debugging the `do_populate_lic` task and I really don't understand how it deduces the relative source directory for the licenses. Adding these absolute paths fixes the issue.

Reply:
Grab the top one from [here](https://github.com/kacf/meta-mender/commits/test), that should fix it. This part should be returned to what it was.

Reply:
Done. Thanks!